redis/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM redis:3.2.8
+COPY redis.conf /usr/local/etc/redis/redis.conf
+CMD [ "redis-server", "/usr/local/etc/redis/redis.conf" ]
```

Comment:
Hi @emanuil-tolev, I'm afraid that is not the case here; this Redis image requires the path to the configuration file to be specified. See the section "Additionally, If you want to use your own redis.conf" in https://hub.docker.com/_/redis/

I have tried placing the redis config file in `/etc/redis.conf` and the result is that it is not read, reverting instead to the default options which will prevent the data from persisting:

```
Attaching to willow_redis_1
redis_1                 | 1:C 26 Apr 10:26:01.147 # Warning: no config file specified, using the default config. In order to specify a config file use redis-server /path/to/redis.conf
redis_1                 |                 _._                                                  
redis_1                 |            _.-``__ ''-._                                             
redis_1                 |       _.-``    `.  `_.  ''-._           Redis 3.2.8 (00000000/0) 64 bit
redis_1                 |   .-`` .-```.  ```\/    _.,_ ''-._                                   
redis_1                 |  (    '      ,       .-`  | `,    )     Running in standalone mode
redis_1                 |  |`-._`-...-` __...-.``-._|'` _.-'|     Port: 6379
redis_1                 |  |    `-._   `._    /     _.-'    |     PID: 1
redis_1                 |   `-._    `-._  `-./  _.-'    _.-'                                   
redis_1                 |  |`-._`-._    `-.__.-'    _.-'_.-'|                                  
redis_1                 |  |    `-._`-._        _.-'_.-'    |           http://redis.io        
redis_1                 |   `-._    `-._`-.__.-'_.-'    _.-'                                   
redis_1                 |  |`-._`-._    `-.__.-'    _.-'_.-'|                                  
redis_1                 |  |    `-._`-._        _.-'_.-'    |                                  
redis_1                 |   `-._    `-._`-.__.-'_.-'    _.-'                                   
redis_1                 |       `-._    `-.__.-'    _.-'                                       
redis_1                 |           `-._        _.-'                                           
redis_1                 |               `-.__.-'                                               
redis_1                 | 
redis_1                 | 1:M 26 Apr 10:26:01.150 # WARNING: The TCP backlog setting of 511 cannot be enforced because /proc/sys/net/core/somaxconn is set to the lower value of 128.
redis_1                 | 1:M 26 Apr 10:26:01.150 # Server started, Redis version 3.2.8
redis_1                 | 1:M 26 Apr 10:26:01.150 # WARNING overcommit_memory is set to 0! Background save may fail under low memory condition. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
```